### DISCUSSION TITLE
How do I run the client without minified javascript?

### DISCUSSION THREAD

**Discussion by th3raid0r** [danny-avila/LibreChat#3591] [Q&A]
As in title - I just need to know the smallest possible change to get a production build to run un-minified. 

It's not anywhere in the docs, presumably because it's using a well known framework. 

Well, I'm having a really difficult time with this. So if y'all can tell me how to get un-minified artifacts, and I'll give ya better bug reports.

**Comment by th3raid0r** [comment]
Also, can we have a debugging.md or troubleshooting.md page for how to fetch any error/log across the stack?

**Comment by danny-avila** [comment]
https://www.librechat.ai/docs/development/get_started

> Use npm run frontend:dev to monitor frontend changes.


`npm run frontend:dev` will populate the source map on your browser, by default on port 3090

**Reply by danny-avila** [reply]
You can use this as long as the vite dev server can access the API server, so it can work whether you run the app locally, on docker, etc.
